app/src/renderer/connectors/node.js

```diff
@@ -24,7 +24,6 @@ module.exports = function (nodeIP, relayPort, mocked = false) {
   }
   // TODO: eventually, get all data from light-client connection instead of RPC
 
-  connector.setMocked(mocked)
-  connector.initRPC(nodeIP)
```

Comment:
why don't we need to call initRPC anymore?

Reply:
init is called only after the main process signals, that it has connected to a node

Reply:
is it because we're using IPC instead?